nixos/modules/services/network-filesystems/glusterfs.nix

```diff
@@ -5,6 +5,22 @@ with lib;
 let
   inherit (pkgs) glusterfs rsync;
 
+  tlsCmd = if cfg.enableTLS then
+  ''
+    mkdir -p /var/lib/glusterd
+    touch /var/lib/glusterd/secure-access
+  ''
+  else
+  ''
+    rm -f /var/lib/glusterd/secure-access
```

Comment:
Ok thanks for clarifying :)

I would imagine instead of putting the private key into the config I would just put a path to a file.
What's the mechanism to make sure this gets symlinked to the correct location in etc? Also via `config.environment.etc`?

Is there a similar mechanism that could be used to put the file into `/var`?